R/rsconnect.R

```diff
@@ -55,15 +55,17 @@
 vetiver_deploy_rsconnect <- function(board, name, version = NULL,
                                      predict_args = list(),
                                      appTitle = glue::glue("{name} model API"),
-                                     ...) {
+                                     ...,
+                                     additional_pkgs = character(0)) {
```

Comment:
I piped this new arg through to the Connect functions, but I did _not_ pipe it through for `vetiver_prepare_docker()`, since this is more like needing to customize the Docker `additional_pkgs` which we don't let you do there. In that situation, we say "use the more modular functions" instead of the higher level one.